### ISSUE TITLE
Streama no change completely the language of interfaz.

### ISSUE THREAD

**Issue by AdrianB1992** [streamaserver/streama#790]
### Issue description

When we edit the language of a user's profile, Streama is translated but not completely.

### Steps to Reproduce

1. Start section with a user.
2. Edit user profile.
3. Change the language to the user.
4. Close session

### Expected Behaviour

Streama should change the interface language, such as the login screen.

### Actual Behaviour

Streama no change complety the language of interfaz.

### Environment Information

- **Operating System**: Ubuntu 18.04 LTS
- **Streama version**: 1.67

**Comment by ImFascinated101** [comment]
By "interfaz", i'm assuming you mean interface?
If that's true, then this isn't a bug of sorts, just needs more stuff to be translated.

**Comment by dularion** [comment]
Agreed with @ImFascinated101 - maybe point out which language and which Texts you are referring to. Maybe with screenshots? It is likely that those texts just havent been translated by the community

**Comment by AdrianB1992** [comment]
I think that in version 1.60 the translation of the main page (where the user and password are entered) is translated correctly.

**Comment by dularion** [comment]
most things are translated though.. see this screenshot of a french translation for example. I only notice a few things, like "Filter by Genre..." and "Search.." 

![image](https://user-images.githubusercontent.com/936076/57476888-db840000-7297-11e9-8914-43ed2563a02d.png)

**Comment by AdrianB1992** [comment]
The part is...

LOGIN: {
			TITLE: 'Introduzca su usuario',
			USERNAME: 'Usuario',
			PASSWORD: 'Contraseña',
			FIRST_TIME_HINT: '¿Es tu primera vez? Prueba \'admin\' en ambos campos.',
			SUBMIT: 'Identificarse',
      SESSION_EXPIRED: 'Su sesión expiró desde su última actividad. Por favor inicie sesión de nuevo.'
		},

**Comment by AdrianB1992** [comment]
Seeing that the file https://github.com/streamaserver/streama/blob/master/grails-app/assets/javascripts/streama/translations/ES_en.js is completely translated, how can I translate streama in full? I would like to contribute in the translations.
